docs/how-to/manage-snapshots.md

```diff
@@ -36,3 +36,28 @@ Deleting old snapshots
 $ fleet snapshot destroy <snapshot_name>
 Snapshot <snapshot_name> is now being destroyed
 ```
+
+Dumping the contents of a snapshot
+----
+
+While you cannot access the contents of a snapshot directly, you can create a new environment using the contents of a snapshot, then dump the database from that environment.
+
+1. Create the new environment from the snapshot.
+
+      ```
+      $ fleet env create <env_name> --snapshot <snapshot_id>
+      ```
+
+2. Dump the contents of the database
```

Comment:
Indeed I can, but I like the source to read with correct numbers as well as the generated site.